### ISSUE TITLE
Discovered server protocol: -1 (unsupported), falling back to 761 (1.19.3)

### ISSUE THREAD

**Issue by Niterux** [Earthcomputer/multiconnect#524]
### Latest Version?

- [X] I confirm I have the latest version of multiconnect

### Description

no matter what I set the version to I always get this message in the console and cannot connect, this s on 1.19.3 with no other mods loaded on quilt loader

joining the server on the versions it suggests works fine

https://user-images.githubusercontent.com/110568869/220003109-682df350-032a-4dc5-8a66-2ffca96abbc3.mp4

### Log File

https://pastebin.com/3evWvh4g

**Comment by Gaming32** [comment]
Yeah, that should be fixed in the current dev builds. Earth hasn't released regular updates recently (or merged PRs, nudge, nudge, Earth), so you'll just have to compile it yourself (the automatic dev builds don't work currently).

**Comment by Niterux** [comment]
any idea on when the next update will be?

**Comment by Gaming32** [comment]
Absolutely no clue. Earth hasn't touched this mod in a while.
